SubmittedAssignments/Shubham_Shinde/Preview.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<!-- <link rel="stylesheet" type="text/css" href="./certificate.css"> -->
+	<link rel="stylesheet" type="text/css" href="./Css/certificate1.css">
+
+	<!-- <link rel="stylesheet" type="text/css" href="./css/bootstrap.min.css"> -->
+</head>
+
+<body>
+	<!--Certificate Wrapper-->
+	<div class="certificate-wrapper">
+
+		<!--Certificate background-->
+		<!-- <div class="certificate-background"> -->
```

Comment:
remove this comment  **class="certificate-background"**

Reply:
done